pkg/apis/kops/componentconfig.go

```diff
@@ -196,6 +196,10 @@ type KubeProxyConfig struct {
 type KubeAPIServerConfig struct {
 	// Image is the docker container used
 	Image string `json:"image,omitempty"`
+	// DisableBasicAuth removes the --basic-auth-file flag
+	DisableBasicAuth bool `json:"disableBasicAuth,omitempty"`
+	// EnableTokenAuth removes the --token-auth-file flag
+	DisableTokenAuth bool `json:"disableTokenAuth,omitempty"`
```

Comment:
Can we get docs for these included in this commit?

Reply:
I've updated the commit to include docs for these two options.